llvm/lib/Transforms/Vectorize/SLPVectorizer.cpp

```diff
@@ -8852,7 +8852,8 @@ BoUpSLP::getEntryCost(const TreeEntry *E, ArrayRef<Value *> VectorizedVals,
       TTI::OperandValueInfo Op1Info = getOperandInfo(E->getOperand(0));
       TTI::OperandValueInfo Op2Info = getOperandInfo(E->getOperand(OpIdx));
       return TTI->getArithmeticInstrCost(ShuffleOrOp, VecTy, CostKind, Op1Info,
-                                         Op2Info) +
+                                         Op2Info, ArrayRef<const Value *>(),
```

Comment:
```suggestion
                                         Op2Info, std::nullopt,
```

Reply:
Applied in the latest commit: (97a2ad9fb87cbb53328936b88341dbd15bde3585).
Thanks for the suggestion and the comments throughout the reviewing process.